onebusaway-android/src/main/java/org/onebusaway/android/ui/ReportStopProblemFragment.java

```diff
@@ -110,16 +111,31 @@ public void onSaveInstanceState(Bundle outState) {
     }
 
     @Override
+    protected Boolean isReportArgumentsValid() {
```

Comment:
@barbeau agreed. Sorry about that, I had a bad experience about primitive types in the past. I am little bit extra careful about this.